base/ltfssaxes.dtx

```diff
@@ -192,503 +250,3177 @@
+\DeclareFontSeriesChangeRule {ex}{sl}   {slex} {}
+\DeclareFontSeriesChangeRule {ex}{sb}   {sbex} {bex}
+\DeclareFontSeriesChangeRule {ex}{b}    {bex}  {}
+\DeclareFontSeriesChangeRule {ex}{eb}   {ebex} {bex}
+\DeclareFontSeriesChangeRule {ex}{ub}   {ubex} {bex}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {ux}{ul}   {ulux} {}
+\DeclareFontSeriesChangeRule {ux}{el}   {elux} {}
+\DeclareFontSeriesChangeRule {ux}{l}    {lux}  {}
+\DeclareFontSeriesChangeRule {ux}{sl}   {slux} {}
+\DeclareFontSeriesChangeRule {ux}{sb}   {sbux} {bux}
+\DeclareFontSeriesChangeRule {ux}{b}    {bux}  {}
+\DeclareFontSeriesChangeRule {ux}{eb}   {ebux} {bux}
+\DeclareFontSeriesChangeRule {ux}{ub}   {ubux} {bux}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {sbuc}{ul} {uluc} {}
+\DeclareFontSeriesChangeRule {sbuc}{el} {eluc} {}
+\DeclareFontSeriesChangeRule {sbuc}{l}  {luc}  {}
+\DeclareFontSeriesChangeRule {sbuc}{sl} {sluc} {}
+\DeclareFontSeriesChangeRule {sbuc}{sb} {sbuc} {}
+\DeclareFontSeriesChangeRule {sbuc}{b}  {buc}  {}
+\DeclareFontSeriesChangeRule {sbuc}{eb} {ebuc} {buc}
+\DeclareFontSeriesChangeRule {sbuc}{ub} {ubuc} {buc}
+\DeclareFontSeriesChangeRule {sbuc}{uc} {sbuc} {}
+\DeclareFontSeriesChangeRule {sbuc}{ec} {sbec} {}
+\DeclareFontSeriesChangeRule {sbuc}{c}  {sbc}  {}
+\DeclareFontSeriesChangeRule {sbuc}{sc} {sbsc} {}
+\DeclareFontSeriesChangeRule {sbuc}{sx} {sbsx} {}
+\DeclareFontSeriesChangeRule {sbuc}{x}  {sbx}  {}
+\DeclareFontSeriesChangeRule {sbuc}{ex} {sbex} {}
+\DeclareFontSeriesChangeRule {sbuc}{ux} {sbux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {sbec}{ul} {ulec} {}
+\DeclareFontSeriesChangeRule {sbec}{el} {elec} {}
+\DeclareFontSeriesChangeRule {sbec}{l}  {lec}  {}
+\DeclareFontSeriesChangeRule {sbec}{sl} {slec} {}
+\DeclareFontSeriesChangeRule {sbec}{sb} {sbec} {bec}
+\DeclareFontSeriesChangeRule {sbec}{b}  {bec}  {}
+\DeclareFontSeriesChangeRule {sbec}{eb} {ebec} {bec}
+\DeclareFontSeriesChangeRule {sbec}{ub} {ubec} {bec}
+\DeclareFontSeriesChangeRule {sbec}{uc} {sbuc} {}
+\DeclareFontSeriesChangeRule {sbec}{ec} {sbec} {}
+\DeclareFontSeriesChangeRule {sbec}{c}  {sbc}  {}
+\DeclareFontSeriesChangeRule {sbec}{sc} {sbsc} {}
+\DeclareFontSeriesChangeRule {sbec}{sx} {sbsx} {}
+\DeclareFontSeriesChangeRule {sbec}{x}  {sbx}  {}
+\DeclareFontSeriesChangeRule {sbec}{ex} {sbex} {}
+\DeclareFontSeriesChangeRule {sbec}{ux} {sbux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {sbc}{ul}  {ulc}  {}
+\DeclareFontSeriesChangeRule {sbc}{el}  {elc}  {}
+\DeclareFontSeriesChangeRule {sbc}{l}   {lc}   {}
+\DeclareFontSeriesChangeRule {sbc}{sl}  {slc}  {}
+\DeclareFontSeriesChangeRule {sbc}{sb}  {sbc}  {}
+\DeclareFontSeriesChangeRule {sbc}{b}   {bc}   {}
+\DeclareFontSeriesChangeRule {sbc}{eb}  {ebc}  {bc}
+\DeclareFontSeriesChangeRule {sbc}{ub}  {ubc}  {bc}
+\DeclareFontSeriesChangeRule {sbc}{uc}  {sbuc} {sbc}
```

Comment:
```suggestion
\DeclareFontSeriesChangeRule {sbc}{uc}  {sbuc} {}
```

The fallback must be a leftover from a previous version. There is no similar rule where the fourth argument is non-empty.